frontend/coprs_frontend/coprs/views/coprs_ns/coprs_general.py

```diff
@@ -451,10 +451,18 @@ def render_copr_integrations(copr, pagure_form):
                   copr.id,
                   copr.webhook_secret) + "<PACKAGE_NAME>/"
 
+    custom_dir_url = "https://{}/webhooks/custom-dir/{}/{}/{}/".format(
+          app.config["PUBLIC_COPR_HOSTNAME"],
+          copr.owner_name,
+          copr.name + ":custom:<SUFFIX>",
+          copr.webhook_secret,
+    ) + "<PACKAGE_NAME>/"
```

Comment:
This should ideally be somewhere in Logic or Models, but it's fine.

Reply:
Yeah, I agree - I just c&p what we already have.